all_msgs/ros2interop/ros-types-all.idl

```diff
@@ -0,0 +1,1825 @@
+module std_msgs {
+  module msg {
+    module dds_ {
+      @appendable
```

Comment:
Why do you need a trailing underscore?

Reply:
This was verified experimentally by working with ROS 2 (mostly RViz) -- connections with DDS applications will be refused on a type name mismatch if the trailing underscores are not present.
(I don't like it very much, but it works).